cachematrix.R

```diff
@@ -1,15 +1,40 @@
-## Put comments here that give an overall description of what your
-## functions do
-
-## Write a short comment describing this function
-
-makeCacheMatrix <- function(x = matrix()) {
-
-}
-
-
-## Write a short comment describing this function
-
-cacheSolve <- function(x, ...) {
-        ## Return a matrix that is the inverse of 'x'
-}
```

Comment:
mycommitment to code

Reply:
This is the Question file 
Please review the commit with the SHA-1 hash db56dc970913c19e4ae59e609c89c49ae0ea81e1